Mathlib/RingTheory/MvPolynomial/MonomialOrder.lean

```diff
@@ -611,6 +613,16 @@ theorem degree_prod [IsCancelMulZero R] {ι : Type*} {P : ι → MvPolynomial σ
   rw [leadingCoeff_ne_zero_iff]
   exact H i hi
 
+lemma degree_mul' [IsCancelMulZero R] [NoZeroDivisors R]
```

Comment:
The changes in this file could be split into a separate PR, for ease of reviewing.